tests/internals/cloudevent_source/cloudevent_source_test.go

```diff
@@ -108,21 +111,43 @@ const (
               cpu: "500m"
   `
 
-	scaledObjectErrTemplate = `
+	scaledObjectTargetNotSupportTemplate = `
 apiVersion: keda.sh/v1alpha1
 kind: ScaledObject
 metadata:
   name: {{.ScaledObject}}
   namespace: {{.TestNamespace}}
 spec:
   scaleTargetRef:
-    name: test
```

Comment:
should we also keep the old test case without `kind` specified to make sure there are no regressions caused by the changes? when the webhooks are disabled, users can still create SO referencing nonexisting target, the operator should handle that gracefully.

Reply:
+1

Reply:
yeah, agree